src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -143,29 +141,34 @@ const Query: FC<PipeProp> = ({Context}) => {
     />
   )
 
-  return useMemo(() => {
-    return (
-      <Context controls={controls}>
-        <Suspense
-          fallback={
-            <SpinnerContainer
-              loading={RemoteDataState.Loading}
-              spinnerComponent={<TechnoSpinner />}
-            />
-          }
-        >
-          <FluxMonacoEditor
-            script={query.text}
-            onChangeScript={updateText}
-            onSubmitScript={() => {}}
-            setEditorInstance={setEditorInstance}
-            wrapLines="on"
-            autogrow
+  return (
+    <Context controls={controls} resizes>
```

Comment:
resizes? that needed?

Reply:
https://github.com/influxdata/ui/pull/2883